packages/forms/src/Testing/TestsForms.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Filament\Forms\Testing;
+
+use Closure;
+use Filament\Forms\ComponentContainer;
+use Filament\Forms\Components\Field;
+use Filament\Forms\Contracts\HasForms;
+use Illuminate\Testing\Assert;
+use Livewire\Testing\TestableLivewire;
+
+/**
+ * @method HasForms instance()
+ *
+ * @mixin TestableLivewire
+ */
+class TestsForms
+{
+    public function assertFormExists(): Closure
+    {
+        return function (string $name = 'form'): static {
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            /** @var ComponentContainer $form */
+            $form = $livewire->$name;
+
+            Assert::assertInstanceOf(
+                ComponentContainer::class,
+                $form,
+                "Failed asserting that a form with the name [{$name}] exists on the [{$livewireClass}] component."
+            );
+
+            return $this;
+        };
+    }
+
+    public function assertFormFieldExists(): Closure
+    {
+        return function (string $fieldName, string $formName = 'form'): static {
+            /** @phpstan-ignore-next-line  */
+            $this->assertFormExists($formName);
+
+            $livewire = $this->instance();
+            $livewireClass = $livewire::class;
+
+            /** @var ComponentContainer $form */
+            $form = $livewire->$formName;
+
+            /** @var ?Field $field */
+            $field = data_get($form->getFlatFields(true), $fieldName, null);
```

Comment:
Please used a named parameter for `getFlatFields(true)`

Reply:
Also this is used in other places, update those also

Reply:
All of these are updated now